isb_web/sqlmodel_database.py

```diff
@@ -109,22 +109,54 @@ def read_things_summary(
 
 
 def get_thing_with_id(session: Session, identifier: str) -> Optional[Thing]:
-    statement = select(Thing).filter(Thing.id == identifier).order_by(Thing.primary_key.asc())
+    statement = (
+        select(Thing).filter(Thing.id == identifier).order_by(Thing.primary_key.asc())
+    )
     result = session.exec(statement).first()
     if result is None:
         # Fall back to querying the Identifiers table
-        join_statement = (
-            select(Thing)
-            .join(ThingIdentifier)
-            .where(ThingIdentifier.guid == identifier)
+        identifiers_statement = select(Thing).where(
+            Thing.identifiers.like(f"%{identifier}%")
```

Comment:
This is the new way to query by identifier -- just a simple `like` statement that seems to be working well.